aldryn_forms/cms_plugins.py

```diff
@@ -398,6 +424,76 @@ def form_post_save(self, instance, form):
             self.send_notification_email(email, form, instance)
 
 
+class FileField(Field):
+    name = _('File upload field')
+
+    model = models.FileUploadFieldPlugin
+
+    form = FileFieldForm
+    form_field = RestrictedFileField
+    form_field_widget = RestrictedFileField.widget
+    form_field_enabled_options = [
+        'label',
+        'help_text',
+        'required',
+        'error_messages',
+        'validators',
+    ]
+    fieldset_general_fields = Field.fieldset_general_fields + [
+        'upload_to',
+        'max_size',
+    ]
+
+    def get_form_field_kwargs(self, instance):
+        kwargs = super(FileField, self).get_form_field_kwargs(instance)
+        if instance.max_size:
+            if 'help_text' in kwargs:
+                kwargs['help_text'] = kwargs['help_text'].replace(
```

Comment:
Because format can fail if the user enters other stuff with braces (the help_text is provided by the end user).